src/data/achievements.js

```diff
@@ -104,6 +104,18 @@ const achievements = [
     reward: state => addItemToInventory(state, cowFeed, reward, true),
   }))(),
 
+  ((reward = 150) => ({
+    id: 'play-during-october',
+    name: 'Halloween Harvest',
+    description: 'Play Farmhand in October and get the gift of the season.',
+    rewardDescription: `${reward} units of ${itemsMap.jackolantern.name}`,
+    condition: () =>
+      // The environment check is necessary to ensure consistent test results.
```

Comment:
this makes me wonder if we should add a `getDate` or `getMonth` utility so we can mock it for tests. what do you think?

Reply:
I think the standard `Date` constructor works well enough for our needs, but there's definitely value in mocking it to avoid special case logic like this. I've mocked out `getMonth` in https://github.com/jeremyckahn/farmhand/commit/b3ba38569db539fc013ad3d5f387db57bda96dae. What do you think about that approach?

Reply:
oops for some reason i left a PR level comment instead of in this thread but i'm good with that!